scriptworker/test/test_gpg.py

```diff
@@ -590,19 +591,24 @@ def test_rebuild_gpg_home_signed(context, trusted_email, tmpdir):
 
 
 # get_git_revision, get_latest_tag {{{1
-# TODO these two tests should use a tarball with a test git repo inside
 @pytest.mark.asyncio
 async def test_get_git_revision():
-    parent_dir = os.path.dirname(__file__)
+    tar = tarfile.open(os.path.join(os.path.dirname(__file__), "data", "test_git_repo.tgz"))
+    tar.extractall(os.path.join(os.path.dirname(__file__), "data"))
+    parent_dir = os.path.join(os.path.dirname(__file__), "data", "testrepo")
     expected = subprocess.check_output(["git", "rev-parse", "HEAD"], cwd=parent_dir)
     assert await sgpg.get_git_revision(parent_dir) == expected.decode('utf-8').rstrip()
+    shutil.rmtree(os.path.join(os.path.dirname(__file__), "data", "testrepo"))
 
 
 @pytest.mark.asyncio
 async def test_get_latest_tag():
-    parent_dir = os.path.dirname(__file__)
+    tar = tarfile.open(os.path.join(os.path.dirname(__file__), "data", "test_git_repo.tgz"))
+    tar.extractall(os.path.join(os.path.dirname(__file__), "data"))
+    parent_dir = os.path.join(os.path.dirname(__file__), "data", "testrepo")
     expected = subprocess.check_output(["git", "describe", "--abbrev=0"], cwd=parent_dir)
```

Comment:
If we hardcode the sha in the previous test, it would make sense to hardcode the tag name 'v2' here.